### DISCUSSION TITLE
Is it possible to get only paragraphs in response?

### DISCUSSION THREAD

**Discussion by vprelovac** [deepgram/community#287] [General help] [answered]
Currently the response includes full transcript with segments etc making the response very large which has bandwidth/latency issues.

I am interested in just the paragrahs for the transcript, is there a way to specify this in the call?

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @vprelovac, I understand that response bodies can be quite large and contain information that is not relevant to you. Our API currently functions so that paragraphs are an addition to the response, and words are always returned as well.

You've likely already considered this, but you can eliminate any other parameters that you don't need that add to the response (such as utterances or diarization).

If you're working with extremely long audio files, you can also consider chunking them (such as 1 hour of audio per request). That's not ideal as then you'll lose context between chunks from the same file, but it may be an acceptable workaround for you.

**Reply by vprelovac** [reply]
Thanks Julia. Yes the issue is that response is 5MB big, where the useful data is only about 10 kilobytes (the actual paragraphs). This causes the response to take several extra seconds to transfer vs being almost instant.

Can you confirm please the optimal parameters to achieve minimal response? I have been using this:

      "model": "nova", 
      "smart_format": True,

**Reply by jkroll-deepgram** [reply]
Hi @vprelovac, apologies for the belated reply. Yes, those parameters will allow you to get smart-formatted transcripts using our top-performing Nova model, without any additional response fields (apart from also adding `paragraphs=true` to get the paragraphs themselves). 

I will pass your feedback along for internal discussion, as you're not the only user who has expressed interest in receiving a subset of our full response content.
